#!/bin/bash

# Hourly datatracker jobs, run as mailman
# 
# This script is expected to be triggered by cron from
# $DTDIR/etc/cron.d/datatracker which should be symlinked from
# /etc/cron.d/

export LANG=en_US.UTF-8
export PYTHONIOENCODING=utf-8

# Make sure we stop if something goes wrong:
program=${0##*/}
trap 'echo "$program($LINENO): Command failed with error code $? ([$$] $0 $*)"; exit 1' ERR

# Note that we're using the last 2.7 release here, not the current release
DTDIR=/a/www/ietf-datatracker/py27
cd $DTDIR/

# Set up the virtual environment
source $DTDIR/env/bin/activate

logger -p user.info -t cron "Running $DTDIR/bin/mm_hourly"

## XXX commented out pending rewrite -- mailman 2 python interface is not available under Python 3
$DTDIR/ietf/manage.py import_mailman_listinfo
